clouddriver/clouddriver-google/src/main/java/com/netflix/spinnaker/clouddriver/google/deploy/handlers/BasicGoogleDeployHandler.java

```diff
@@ -1079,34 +1143,47 @@ private void updateRegionalBackendServices(
             || !lbInfo.internalHttpLoadBalancers.isEmpty())) {
       regionBackendServicesToUpdate.forEach(
           backendService -> {
-            safeRetry.doRetry(
-                updateBackendServices(
-                    description.getCredentials().getCompute(),
-                    description.getCredentials().getProject(),
-                    backendService.getName(),
-                    backendService,
-                    region),
-                "Internal load balancer backend service",
-                task,
-                List.of(400, 412),
-                Collections.emptyList(),
-                Map.of(
-                    "action",
-                    "update",
-                    "phase",
-                    BASE_PHASE,
-                    "operation",
-                    "updateRegionBackendServices",
-                    TAG_SCOPE,
-                    SCOPE_REGIONAL,
-                    TAG_REGION,
-                    region),
-                registry);
+            Operation backendServiceOperation =
+                (Operation)
+                    safeRetry.doRetry(
+                        updateBackendServices(
+                            description.getCredentials().getCompute(),
+                            description.getCredentials().getProject(),
+                            backendService.getName(),
+                            backendService,
+                            region),
+                        "Internal load balancer backend service",
+                        task,
+                        List.of(400, 412),
+                        Collections.emptyList(),
+                        Map.of(
+                            "action",
+                            "update",
+                            "phase",
+                            BASE_PHASE,
+                            "operation",
+                            "updateRegionBackendServices",
+                            TAG_SCOPE,
+                            SCOPE_REGIONAL,
+                            TAG_REGION,
+                            region),
+                        registry);
             task.updateStatus(
                 BASE_PHASE,
                 String.format(
                     "Done associating server group %s with backend service %s.",
                     serverGroupName, backendService.getName()));
+
```

Comment:
This doesn't seem to quite match the other PR where the status was set to "Waiting for backend service..."?